README.md

```diff
@@ -1710,6 +1710,12 @@ _Libraries for working with Microsoft Excel._
 - [xlsx](https://github.com/tealeg/xlsx) - Library to simplify reading the XML format used by recent version of Microsoft Excel in Go programs.
 - [xlsx](https://github.com/plandem/xlsx) - Fast and safe way to read/update your existing Microsoft Excel files in Go programs.
 
+### Microsoft Word
+
+_Libraries for working with Microsoft Word._
+
+- [godocx](https://github.com/gomutex/godocx) - Golang library for reading and writing Microsoft Word (Docx) files. 
```

Comment:
No need to mention that it's a Go library, because the entire repo is just for Go projects.

```diff
+ Golang library for reading and writing Microsoft Word (Docx) files. 
- Library for reading and writing Microsoft Word (Docx) files. 
```